rules/cross-platform/guided_onboarding_sample_rule.toml

```diff
@@ -15,18 +15,8 @@ enabled = false
 false_positives = [
     "This rule is not looking for threat activity. Disable the rule if you're already familiar with alerts.",
 ]
-from = "now-24h"
-index = [
-    "apm-*-transaction*",
-    "auditbeat-*",
-    "endgame-*",
-    "filebeat-*",
-    "logs-*",
-    "packetbeat-*",
-    "traces-apm*",
-    "winlogbeat-*",
-    "-*elastic-cloud-logs-*",
-]
+from = "now-1h"
```

Comment:
It works, but it is strange

![image](https://github.com/elastic/detection-rules/assets/26856693/8be931f7-eb51-4fed-bd0f-3f8968f98c7f)